flask/flask-common/src/main/java/net/corda/flask/common/LockFile.java

```diff
@@ -17,7 +17,6 @@
 
     private static RandomAccessFile createLockFile(Path path) throws IOException {
         Files.createDirectories(path.getParent());
-        if (!Files.exists(path)) Files.createFile(path);
         return new RandomAccessFile(path.toFile(), "rw");
```

Comment:
How did you verify that this is actually safe?

Reply:
what do you mean with "safe"? safe that the following line will create the file when it doesn't exist or safe that a race condition won't crash the code?

Reply:
at the moment we have unit test creating a flask jar and running it in a fresh environment, but we do not have a test for race conditions (as they are quite tricky to reproduce even in testing environment)